src/frontend/csrs-portal/src/app/components/landing/landing.component.ts

```diff
@@ -77,61 +66,18 @@ export class LandingComponent implements OnInit
 
           this.logInOutService.currentUser(isAuthenticated);
       });
-
-
     }
```

Comment:
Please in future make sure the bracket is aligned right. Otherwise it was giving the impression of extraneous bracket.

Reply:
bracket was aligned right